python/ray/util/client/server/dataservicer.py

```diff
@@ -115,6 +119,47 @@ def Datapath(self, request_iterator, context):
                             req.prep_runtime_env)
                         resp = ray_client_pb2.DataResponse(
                             prep_runtime_env=resp_prep)
+                elif req_type == "task":
+                    with self.clients_lock:
+                        resp_ticket = self.basic_service.Schedule(req.task)
+                        resp = ray_client_pb2.DataResponse(
+                            task_ticket=resp_ticket)
+                elif req_type == "terminate":
+                    with self.clients_lock:
+                        response = self.basic_service.Terminate(req.terminate)
+                        resp = ray_client_pb2.DataResponse(terminate=response)
+                elif req_type == "list_named_actors":
+                    with self.clients_lock:
+                        response = self.basic_service.ListNamedActors(
+                            req.list_named_actors)
+                        resp = ray_client_pb2.DataResponse(
+                            list_named_actors=response)
+                elif req_type == "cluster_info":
+                    with self.clients_lock:
+                        response = self.basic_service.ClusterInfo(
+                            req.cluster_info)
+                        resp = ray_client_pb2.DataResponse(
+                            cluster_info=response)
+                elif req_type == "kv_get":
+                    with self.clients_lock:
+                        response = self.basic_service.KVGet(req.kv_get)
+                        resp = ray_client_pb2.DataResponse(kv_get=response)
+                elif req_type == "kv_exists":
+                    with self.clients_lock:
+                        response = self.basic_service.KVExists(req.kv_exists)
+                        resp = ray_client_pb2.DataResponse(kv_exists=response)
+                elif req_type == "kv_put":
+                    with self.clients_lock:
+                        response = self.basic_service.KVPut(req.kv_put)
+                        resp = ray_client_pb2.DataResponse(kv_put=response)
+                elif req_type == "kv_del":
+                    with self.clients_lock:
+                        response = self.basic_service.KVDel(req.kv_del)
+                        resp = ray_client_pb2.DataResponse(kv_del=response)
+                elif req_type == "kv_list":
+                    with self.clients_lock:
+                        response = self.basic_service.KVList(req.kv_list)
+                        resp = ray_client_pb2.DataResponse(kv_list=response)
```

Comment:
are these public APIs? if not, why do we need them?

Reply:
They are the handlers for the newly added interface. Will remove them for now.